style.css

```diff
@@ -1,615 +1,1361 @@
-  align-items: center;
-  margin-top: 27px;
-  background-color: #f9f9fa;
-}
-
-.contact-top-img {
-  width: 327px;
-  height: 74px;
-  background: url('assets/Shape.png') no-repeat, url('assets/Shape-2.png') no-repeat 10% 12%;
-}
-
-.contact h2 {
-  margin-top: 32px;
-  margin-bottom: 0;
-  font-family: 'Playfair Display', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 50px;
-  line-height: 100%;
-  color: #1a2236;
-}
-
-.contact p {
-  margin-top: 20px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-  line-height: 21px;
-  letter-spacing: -0.01em;
-  color: #1a2236;
-}
-
-.contact-form {
-  display: flex;
-  flex-direction: column;
-  align-content: center;
-  margin-left: 18px;
-}
-
-.contact-form > label {
-  width: 72px;
-  height: 15px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 10px;
-  line-height: 15px;
-  color: #7d7c84;
-  margin-bottom: 3px;
-}
-
-/* please put the styles of the inputs, textareas and button */
-
-#name {
-  box-sizing: border-box;
-  width: 288px;
-  height: 52px;
-  background-color: #fff;
-  border: 1px solid #dfdfe0;
-  border-radius: 4px;
-  margin-bottom: 27px;
-}
-
-#name::placeholder {
-  width: 74px;
-  height: 21px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-  line-height: 21px;
-  color: #1a2236;
-}
-
-#email {
-  box-sizing: border-box;
-  width: 288px;
-  height: 52px;
-  background-color: #fff;
-  border: 1px solid #1a2236;
-  border-radius: 4px;
-  margin-bottom: 29px;
-}
-
-#email::placeholder {
-  width: 138px;
-  height: 24px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  color: #1a2236;
-}
-
-#message {
-  box-sizing: border-box;
-  width: 288px;
-  height: 112px;
-  background-color: #fff;
-  border: 1px solid #dfdfe0;
-  border-radius: 4px;
-  margin-bottom: 20px;
-}
-
-#message::placeholder {
-  width: 184px;
-  height: 21px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-  line-height: 21px;
-  color: #7d7c84;
-}
-
-.links {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  margin-top: 31px;
-  height: 137px;
-  background-color: #1a2236;
-}
-
-.resume {
-  padding-top: 28px;
-  padding-left: 79px;
-  margin-bottom: 0;
-}
-
-.resume a {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
-  color: #92dcdc;
-  text-decoration: none;
-}
-
-.social-media-icons {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  height: 24px;
-  padding-left: 0;
-  padding-top: 27.25px;
-  margin-top: 0;
+@media screen and (max-width: 767px) {
+  html,
+  body {
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+  }
+
+  .mobile-version {
```

Comment:
- [ ] please I would suggest you to kindly stick to the figma template the design must look exact like the figma template according to requirement
- [ ] your design

![Screenshot_20230505-201516](https://user-images.githubusercontent.com/92458236/236550781-80def8a3-8dd7-4ddd-ac5e-6a59debfc4ee.jpg)

- [ ] figma design
![Screenshot_20230505-185631_1](https://user-images.githubusercontent.com/92458236/236551151-0d1a31dd-04d2-4784-a9fc-55d9b76ab134.jpg)